cmd/cstor-pool-mgmt/pool/pool_topology.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2018 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pool
+
+import (
+	"encoding/json"
+	"github.com/golang/glog"
+)
+
+// Topology contains the topology strucure of disks used in backend
+type Topology struct {
+	Children int      `json:"vdev_children,omitempty"`
```

Comment:
it will be good to have some code comments for these fields

Reply:
added